.eslintrc.yml

```diff
@@ -28,7 +28,7 @@ rules:
   no-alert: warn
   no-mixed-operators: warn
   no-param-reassign: warn
-  no-plusplus: warn
+  no-plusplus: off
```

Comment:
Why would we want this? It ends up highlighting `for` loops.

Reply:
It's from the [Airbnb style guide](https://github.com/airbnb/javascript#variables--unary-increment-decrement), included by the `airbnb` preset.

My approach to the eslint file was to use the Airbnb preset, and then override its errors with warnings to buy us time to fix them.

I don't have a strong opinion on disabling particular warnings. [ESlint's rationale for this one](https://eslint.org/docs/rules/no-plusplus) seems weak, given that the code is being babelized anyway. 

That said, a weaker alternative to `off` is `[warn, allowForLoopAfterthoughts: true]` “allows unary operators `++` and `--` in the afterthought (final expression) of a `for` loop."